tasks/admin.py

```diff
@@ -1,12 +1,16 @@
 from django.contrib import admin
 from tasks.models import Task, TaskSet, Stalker, Active
 
+from easy_select2 import select2_modelform
+
 
 class TaskAdmin(admin.ModelAdmin):
     list_display = ['id', 'title', 'task_set', 'public', 'type']
     search_fields = ['id', 'title', 'text', 'example_solution']
     list_filter = ['type', 'public', 'task_set']
 
+    form = select2_modelform(Task)
```

Comment:
No dobre, aj v M-M blika kurzor + pridavanie help textu by sa asi robilo zbytocne komplikovane,
tak to necham tak ...